Nodejs/Product/Nodejs/ProjectTemplates/AzureExpress4App/typings.json

```diff
@@ -5,7 +5,6 @@
     "debug": "registry:dt/debug#0.0.0+20160317120654",
     "express": "registry:dt/express#4.0.0+20160317120654",
     "express-serve-static-core": "registry:dt/express-serve-static-core#0.0.0+20160602151406",
-    "jade": "registry:dt/jade#0.0.0+20160316155526",
```

Comment:
This was removed because the pug type definitions only exist in the npm registry, not in dt. Multiple registries (and ones that are not `--global`) are not something our super simplified typings acquisition system can handle. VS15 should handle typings better.

For the majority of use cases, users are not invoking pug directly but just using it through express.